packages/react-components/react-components/package.json

```diff
@@ -49,7 +49,7 @@
     "@fluentui/react-link": "^9.0.9",
     "@fluentui/react-menu": "^9.3.1",
     "@fluentui/react-overflow": "9.0.0-beta.12",
-    "@fluentui/react-persona": "^9.0.3",
+    "@fluentui/react-persona": "^9.1.0-beta.0",
```

Comment:
prereleases need to use pinned version only otherwise it will resolve in version that customers/us might not expect.